extensions/ql-vscode/src/extension.ts

```diff
@@ -177,9 +178,12 @@ function getCommands(
     "codeQL.restartQueryServer": async () =>
       withProgress(
         async (progress: ProgressCallback, token: CancellationToken) => {
-          // We restart the CLI server too, to ensure they are the same version
+          // Restart all of the spawned servers: cli, query, and language.
           cliServer.restartCliServer();
-          await queryRunner.restartQueryServer(progress, token);
+          await Promise.all([
+            queryRunner.restartQueryServer(progress, token),
+            ideServer.restart(),
```

Comment:
Is this calling this method — https://github.com/microsoft/vscode-languageserver-node/blob/b491193996e01435e44b00d677516fa4449bee82/client/src/node/main.ts#L190?

Reply:
That's right.